WcaOnRails/app/views/competitions/_index_competitions_list.html.erb

```diff
@@ -50,4 +43,35 @@
       });
     });
   </script>
+<% else %>
+  <% if current_user&.can_see_admin_competitions? %>
+    <% if competitions.empty? %>
+      <%= render 'no_competitions_found' %>
+    <% else %>
+      <% past_competitions, upcoming_competitions = competitions.partition(&:is_over?) %>
+      <% recent_competitions, distant_competitions = past_competitions.partition { |c| (Date.today - c.end_date).to_i < 30 } %>
+      <% unless upcoming_competitions.empty? %>
+        <div class="col-md-12" id="upcoming-comps">
+          <%= render 'admin_index_table', competitions: upcoming_competitions, title: "Upcoming competitions" %>
+        </div>
+      <% end %>
+      <div class="col-md-12" id="recent-comps">
+        <%= render 'admin_index_table', competitions: recent_competitions, title: "Recent competitions" %>
```

Comment:
I like being explicit... how do you feel about saying "Competitions within the last 30 days"? Then you can use this new constant I asked you to create above =)